src/app/containers/pleiepenger/Fordeling/Komponenter/SokersBarn.tsx

```diff
@@ -0,0 +1,189 @@
+    sokersIdent: string;
+    barnetHarInteFnrFn?: (harBarnetFnr: boolean) => void;
+    erBarnUtdatert?: boolean;
+    riktigIdentIJournalposten?: JaNei;
+}
+
+type ISokersBarnProps = WrappedComponentProps &
+    ISokersBarnStateProps &
+    ISokersBarnDispatchProps &
+    ISokersBarn;
+
+const SokersBarnComponent: React.FunctionComponent<ISokersBarnProps> = (
+    props: ISokersBarnProps
+) => {
+    const {
+        intl,
+        barnetHarInteFnrFn,
+        journalpost,
+        identState,
+        erBarnUtdatert,
+        riktigIdentIJournalposten,
+        sokersIdent
+    } = props;
+
+    const journalpostident = journalpost?.norskIdent;
+    const finnesRiktigIdentIJournalposten = typeof riktigIdentIJournalposten !== 'undefined';
+    const skalBarnetHarIkkeFnrCheckboksVises = typeof barnetHarInteFnrFn !== 'undefined';
+
+    const [barnetsIdent, setBarnetsIdent] = useState<string>('');
+    const [barnetHarIkkeFnr, setBarnetHarIkkeFnr] = useState<boolean>(false);
+    const [gjelderAnnetBarn, setGjelderAnnetBarn] = useState<boolean>(false);
+
+
+    useEffect(() => {
+        if(!!journalpost && typeof journalpost?.norskIdent !== 'undefined') {
+            props.hentBarn(journalpost?.norskIdent);
+        }
+    }, []);
+
+    const barnetsIdentInputFieldOnChange = (event: any) => {
+        setBarnetsIdent(event.target.value.replace(/\D+/, ''));
+        setIdentAction(identState.ident1, event.target.value)
+    }
+
+    const barnetsIdentInputFieldOBlur = (event: any) => {
+        props.setIdentAction(finnesRiktigIdentIJournalposten && riktigIdentIJournalposten === JaNei.JA ? (journalpostident || '') : sokersIdent, event.target.value, identState.annenSokerIdent);
+    }
+
+    const nullUtBarnetsIdent = () => {
+        setBarnetsIdent('');
+        setIdentAction(identState.ident1, '');
+        props.setIdentAction(identState.ident1, '', identState.annenSokerIdent);
+    }
+
+    const barnHarIkkeFnrCheckboks = (checked: boolean) => {
+        setBarnetHarIkkeFnr(checked);
+        skalBarnetHarIkkeFnrCheckboksVises && barnetHarInteFnrFn(checked);
+        if (checked) {
+            setBarnetsIdent('');
+            props.setIdentAction(finnesRiktigIdentIJournalposten && riktigIdentIJournalposten === JaNei.JA ? (journalpostident || '') : sokersIdent, null);
+        }
+    }
+
+    return (<div className="sokersBarn">
+        {(typeof erBarnUtdatert !== 'undefined' && !erBarnUtdatert || typeof erBarnUtdatert === 'undefined') && !!props.fellesState.hentBarnSuccess && typeof props.fellesState.barn !== 'undefined' && props.fellesState.barn.length > 0 && <>
+          <Select
+            value={barnetsIdent}
+            bredde="l"
+            label={intlHelper(intl, 'ident.identifikasjon.velgBarn')}
+            onChange={barnetsIdentInputFieldOnChange}
+            disabled={gjelderAnnetBarn}
+            onBlur={barnetsIdentInputFieldOBlur}
+          >
+            <option key={uuidv4()} value={""}>
```

Comment:
denne gir deg random ID hver gang, så den bør ikke brukes som key. Keyen må være stabil for at ikke React skal fjerne elementet og lage det på nytt hver gang